### PR TITLE
Update Pages bindings to reflect new dashboard, correct queues documentation, and add Hyperdrive details

### PR DESCRIPTION
**Pull request by thomasgauvin**
This updates the docs to reflect the new dashboard.

Also some fixes & updates to Queues and Hyperdrive.

### CODE REVIEW

File: src/content/docs/pages/functions/bindings.mdx [hunk 1]
```diff
@@ -296,9 +296,10 @@ To configure a Vectorize index binding via the Cloudflare dashboard:
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
 3. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
-4. Select your Pages project > **Settings** > **Functions** > **Vectorize index bindings** > **Add binding**.
+4. Select your Pages project > **Settings**.
+4. Select your Pages environment > **Bindings** > **Add** > **Vectorize index**.
```
Comment (Oxyjun) [inline src/content/docs/pages/functions/bindings.mdx]: Does the numbering here need to be fixed? Seems like there's two `step 4`s

File: src/content/docs/pages/functions/bindings.mdx [hunk 2]
```diff
@@ -27,11 +27,11 @@ To configure a KV namespace binding via the Cloudflare dashboard:
...
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
-3. Select your Pages project > **Settings** > **Functions** > **KV namespace bindings** > **Add binding**.
-4. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
-5. Give your binding a name under **Variable name**.
-6. Under **KV namespace**, select your desired namespace. You must repeat steps 5 and 6 for both the **Production** and **Preview** environments.
-7. Redeploy your project for the binding to take effect.
+3. Select your Pages project > **Settings**.
+4. Select your Pages environment > **Bindings** > **Add** > **KV namespace**.
+4. Give your binding a name under **Variable name**.
```
Comment (Oxyjun) [inline src/content/docs/pages/functions/bindings.mdx]: Numbering may need to be fixed

File: src/content/docs/pages/functions/bindings.mdx [hunk 3]
```diff
@@ -797,8 +816,9 @@ To add secrets to your Pages project:
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
-3. Select your Pages project > select **Settings** > **Environment variables**.
-4. Selecting **Add variables** under **Production** and/or **Preview**.
+3. Select your Pages project > select **Settings**.
+4. Select your Pages environment > **Variables and Secrets** > **Add**.
+5. Set a variable name and value.
```
Comment (Oxyjun) [inline src/content/docs/pages/functions/bindings.mdx:821]: Numbering? Seems like there are two `step 5`s

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/pages/functions/bindings.mdx [hunk 4]
```diff
@@ -27,11 +27,11 @@ To configure a KV namespace binding via the Cloudflare dashboard:
...
-3. Select your Pages project > **Settings** > **Functions** > **KV namespace bindings** > **Add binding**.
-4. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
-5. Give your binding a name under **Variable name**.
-6. Under **KV namespace**, select your desired namespace. You must repeat steps 5 and 6 for both the **Production** and **Preview** environments.
-7. Redeploy your project for the binding to take effect.
+3. Select your Pages project > **Settings**.
+4. Select your Pages environment > **Bindings** > **Add** > **KV namespace**.
+4. Give your binding a name under **Variable name**.
+5. Under **KV namespace**, select your desired namespace.
+6. Redeploy your project for the binding to take effect.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
5. Give your binding a name under **Variable name**.
6. Under **KV namespace**, select your desired namespace.
7. Redeploy your project for the binding to take effect.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 5]
```diff
@@ -797,8 +816,9 @@ To add secrets to your Pages project:
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
-3. Select your Pages project > select **Settings** > **Environment variables**.
-4. Selecting **Add variables** under **Production** and/or **Preview**.
+3. Select your Pages project > select **Settings**.
+4. Select your Pages environment > **Variables and Secrets** > **Add**.
+5. Set a variable name and value.
 5. Select **Encrypt** to create your secret.
 6. Select **Save**.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
3. Select your Pages project > select **Settings**.
4. Select your Pages environment > **Variables and Secrets** > **Add**.
5. Set a variable name and value.
6. Select **Encrypt** to create your secret.
7. Select **Save**.
```

File: src/content/docs/pages/functions/bindings.mdx (hunk 1 above)
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
5. Select your Pages environment > **Bindings** > **Add** > **Vectorize index**.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 6]
```diff
@@ -296,9 +296,10 @@ To configure a Vectorize index binding via the Cloudflare dashboard:
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
 3. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
-4. Select your Pages project > **Settings** > **Functions** > **Vectorize index bindings** > **Add binding**.
+4. Select your Pages project > **Settings**.
+5. Select your Pages environment > **Bindings** > **Add** > **Vectorize index**.
 5. Give your binding a name under **Variable name**.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
6. Give your binding a name under **Variable name**.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 7]
```diff
@@ -296,9 +296,10 @@ To configure a Vectorize index binding via the Cloudflare dashboard:
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
 3. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
-4. Select your Pages project > **Settings** > **Functions** > **Vectorize index bindings** > **Add binding**.
-5. Give your binding a name under **Variable name**.
-6. Under **Vectorize index**, select your desired Vectorize index. You must repeat steps 5 and 6 for both the **Production** and **Preview** environments.
+4. Select your Pages project > **Settings**.
+5. Select your Pages environment > **Bindings** > **Add** > **Vectorize index**.
+6. Give your binding a name under **Variable name**.
+6. Under **Vectorize index**, select your desired Vectorize index.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
7. Under **Vectorize index**, select your desired Vectorize index.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 8]
```diff
@@ -296,9 +296,10 @@ To configure a Vectorize index binding via the Cloudflare dashboard:
...
 2. In **Account Home**, select **Workers & Pages**.
 3. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
-4. Select your Pages project > **Settings** > **Functions** > **Vectorize index bindings** > **Add binding**.
-5. Give your binding a name under **Variable name**.
-6. Under **Vectorize index**, select your desired Vectorize index. You must repeat steps 5 and 6 for both the **Production** and **Preview** environments.
+4. Select your Pages project > **Settings**.
+5. Select your Pages environment > **Bindings** > **Add** > **Vectorize index**.
+6. Give your binding a name under **Variable name**.
+7. Under **Vectorize index**, select your desired Vectorize index.
 7. Redeploy your project for the binding to take effect.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
8. Redeploy your project for the binding to take effect.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 9]
```diff
@@ -505,10 +506,10 @@ To configure a Service binding via the Cloudflare dashboard:
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
-3. Select your Pages project > **Settings** > **Functions** > **Service bindings** > **Add binding**.
-4. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
+3. Select your Pages project > **Settings**.
+4. Select your Pages environment > **Functions** > **Add** > **Service binding**.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
4. Select your Pages environment > **Bindings** > **Add** > **Service binding**.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 10]
```diff
@@ -86,10 +86,10 @@ To configure a Durable Object binding via the Cloudflare dashboard:
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
-3. Select your Pages project > **Settings** > **Functions** > **Durable Object bindings** > **Add binding**.
-4. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
+3. Select your Pages project > **Settings**.
+4. Select your Pages environment > **Bindings** > **Add binding** > **Durable Object**.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
4. Select your Pages environment > **Bindings** > **Add** > **Durable Object**.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 11]
```diff
@@ -149,10 +149,10 @@ To configure a R2 bucket binding via the Cloudflare dashboard:
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
-3. Select your Pages project > **Settings** > **Functions** > **R2 bucket bindings** > **Add binding**.
-4. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
+3. Select your Pages project > **Settings**.
+4. Select your Pages environment > **Bindings** > **Add binding** > **R2 bucket**.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
4. Select your Pages environment > **Bindings** > **Add** > **R2 bucket**.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 12]
```diff
@@ -220,10 +220,10 @@ To configure a D1 database binding via the Cloudflare dashboard:
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
 2. In **Account Home**, select **Workers & Pages**.
-3. Select your Pages project > **Settings** > **Functions** > **D1 database bindings** > **Add binding**.
-4. Choose whether you would like to set up the binding in your **Production** or **Preview** environment.
+3. Select your Pages project > **Settings**.
+4. Select your Pages environment > **Bindings** > **Add binding**> **D1 database bindings**.
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
4. Select your Pages environment > **Bindings** > **Add**> **D1 database bindings**.
```

File: src/content/docs/pages/functions/bindings.mdx [hunk 13]
```diff
@@ -668,7 +683,11 @@ export const onRequest: PagesFunction<Env> = async (context) => {
 };
 ```
 
-</TabItem> </Tabs>
+</TabItem> </Tabs> 
+
+### Interact with your Hyperdrive binding locally
+
+To interact with your Hyperdrive binding locally, use the `wrangler.toml` file to configure your Hyperdrive binding. Then, run [`npx wrangler pages dev <OUTPUT_DIR>`](/workers/wrangler/commands/#dev-1).
```
Comment (thomasgauvin) [inline src/content/docs/pages/functions/bindings.mdx]:
```suggestion
To interact with your Hyperdrive binding locally, you must provide a local connection string to your database that your Pages project will connect to directly. You can set an environment variable `WRANGLER_HYPERDRIVE_LOCAL_CONNECTION_STRING_<BINDING_NAME>` with the connection string of the database, or use the `wrangler.toml` file to configure your Hyperdrive binding with a `localConnectionString` as specified in [Hyperdrive documentation for local development](/hyperdrive/configuration/local-development/). Then, run [`npx wrangler pages dev <OUTPUT_DIR>`](/workers/wrangler/commands/#dev-1).
```

**Pull request merged**